tests/components/SearchBar.stories.tsx

```diff
@@ -61,12 +48,24 @@ export const Primary = (args: SearchBarProps) => {
 export const DropdownExpanded = Primary.bind({});
 DropdownExpanded.play = ({ canvasElement }) => {
   const canvas = within(canvasElement);
-  userEvent.click(canvas.getByRole('textbox'));
+  const textboxEl = canvas.getByRole('textbox');
+  conductRecentSearches(textboxEl);
+  userEvent.click(textboxEl);
 };
 
 export const DropdownHighlight = Primary.bind({});
 DropdownHighlight.play = ({ canvasElement }) => {
   const canvas = within(canvasElement);
-  userEvent.click(canvas.getByRole('textbox'));
+  const textboxEl = canvas.getByRole('textbox');
+  conductRecentSearches(textboxEl);
```

Comment:
previously this didn't work, but I think it's because we needed to have a delay before hitting the Tabs